pkgs/development/python-modules/pikepdf/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, buildPythonPackage, python3Packages, fetchPypi, qpdf, pythonOlder }:
+
+
+buildPythonPackage rec {
+  pname = "pikepdf";
+  version = "1.2.0";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1rhp66j091px0jvlx817h8csadkmvg26l3y98ccs5cd06xh6ly2p";
+  };
+
+  buildInputs = with python3Packages; [ defusedxml lxml qpdf pybind11
+                                        setuptools setuptools_scm
+                                      ];
+  checkInputs = with python3Packages; [ nose pytest hypothesis pillow ];
```

Comment:
Oh actually, don't take dependencies from pythonPackages at all, put all these packages in the argument list instead, see how other packages in nixpkgs do this.

Reply:
This is the error you get if you don't have this:

```bash
$ nix-shell -p 'python3.withPackages (p: [ p.pikepdf ])' -I nixpkgs=$PWD --pure --run 'python3 -c "import pikepdf"'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/nix/store/697ghwi4qfnl90r1km0566bjcqhjgn3c-python3-3.7.3-env/lib/python3.7/site-packages/pikepdf/__init__.py", line 24, in <module>
    from .models import (
  File "/nix/store/697ghwi4qfnl90r1km0566bjcqhjgn3c-python3-3.7.3-env/lib/python3.7/site-packages/pikepdf/models/__init__.py", line 11, in <module>
    from .metadata import PdfMetadata
  File "/nix/store/697ghwi4qfnl90r1km0566bjcqhjgn3c-python3-3.7.3-env/lib/python3.7/site-packages/pikepdf/models/metadata.py", line 16, in <module>
    from defusedxml.lxml import parse
ModuleNotFoundError: No module named 'defusedxml'
```